src/views/tutorial-view/server.ts

```diff
@@ -60,6 +61,10 @@ export async function getTutorialPageProps(
     collection.data,
     fullTutorialData.collectionCtx
   )
+  const sidebarSections = await getCollectionViewSidebarSections(
```

Comment:
We should add a note here re: that this calls _all_ product collections and how we plan to potentially refactor to load that data clientside.

Reply:
Added in https://github.com/hashicorp/dev-portal/pull/351/commits/cf4703efef38589eaf7c4cd7c2bb2718f25684f8!